man/rmdhunks/vcr-design.Rmd

```diff
@@ -0,0 +1,152 @@
+## Design
+
+This section explains `vcr`'s internal design and architecture.
+
+## Where vcr comes from and why R6
```

Comment:
Link to something explaining R6? https://adv-r.hadley.nz/r6.html?

Reply:
yes probably that

Reply:
aldready linking to it in the paragraph